assets/package.json

```diff
@@ -15,7 +15,7 @@
   "prettier": "@nulib/prettier-config",
   "dependencies": {
     "@absinthe/socket-apollo-link": "^0.2.1",
-    "@apollo/client": "latest",
+    "@apollo/client": "*",
```

Comment:
@mathewjordan Is this intended?

Reply:
Not sure how that was included. Definitely not intended. Is `latest` the same as `*`?

Reply:
I think the `*` means any version?  Let's revert this back to `latest`, then merge it in.